tests/integration/oauth_test.go

```diff
@@ -477,3 +479,24 @@ func TestOAuthIntrospection(t *testing.T) {
 	resp = MakeRequest(t, req, http.StatusUnauthorized)
 	assert.Contains(t, resp.Body.String(), "no valid authorization")
 }
+
+func TestGitOpWithOAuthDisabled(t *testing.T) {
```

Comment:
What does it test? It tests nothing?

Reply:
ee7116a7fea40a8ccfd2f18e9a2de5585517f766

Reply:
Still, what does it test? If revert all other code, the test still passes? And I guess such "basic auth" should already have been tested somewhere else?